packages/common/src/utils/findAnalyticsMatch.ts

```diff
@@ -15,7 +15,7 @@
  */
 
 /**
- * Internal dependencies.
+ * External dependencies.
```

Comment:
```suggestion
 * Internal dependencies.
```

Reply:
Resolved in [57583d3](https://github.com/GoogleChromeLabs/ps-analysis-tool/pull/436/commits/57583d3ed20b80150a0731b95db7e2ffcd3b98ce)